docs/_includes/generic-content.html

```diff
@@ -0,0 +1,9 @@
+<section class="content_main">
```

Comment:
Does this have an element with a `content` class as a parent container to this? It should.